modules/imap/hm-ews.php

```diff
@@ -42,7 +42,12 @@ class Hm_EWS {
 
     public function connect(array $config) {
         try {
-            $this->ews = ExchangeWebServices::fromUsernameAndPassword($config['server'], $config['username'], $config['password'], ['version' => ExchangeWebServices::VERSION_2016, 'trace' => 1]);
+            $password = $config['password'] ?? '';
```

Comment:
Why would the config be in such an unexpected format? Isn't it written by cypht? Is it probably an old version of cypht that is not properly upgraded? We should find out and keep the config file properly encoded - i.e. servers array having the right keys...